web/src/main/webapp/ui/menu.ts

```diff
@@ -249,10 +372,34 @@ export class ContextMenu extends BaseMenu<MenuItem> implements IHtmlElement {
     public setAction(mi: MenuItem, enabled: boolean): void {
         const index = this.find(mi.text);
         const cell = this.cells[index];
-        if (mi.action != null && enabled)
-            cell.onclick = () => { this.hide(); mi.action(); };
-        else
+        if (mi.action != null && enabled) {
+            if (cell.classList.contains("expandableMenu")) {
+              cell.onclick = () => {
```

Comment:
I thought you said that this is hover. Where is the hover check?

Reply:
I see the hover handled with onmouseenter. But why is this necessary?

Reply:
If I click on the menu item it does not disappear; in fact, the whole menu stays resident after the action, and there seems to be no way to remove the menu.

Reply:
Ideally, we want the `expandableMenu` to expand the submenus only, without having another onclick() task. The main purpose of this code is to prevent `this.hide();` because the menu should stay still while the submenus are being displayed. Do you agree if I remove the setAction for `expandableMenu`? The onmouseenter will display the submenus